erts/doc/src/erlang.xml

```diff
@@ -1927,16 +1927,23 @@ end</code>
 	  channel.
 	</p>
 	<note><p>
-	  Only the process registered as distribution
-	  controller for the distribution channel identified by
-	  <c><anno>DHandle</anno></c> is allowed to call this
-	  function.
+          When the distribution controller for the distribution
+          channel identified by <c><anno>DHandle</anno></c> is a
+          process, it is the only process allowed to call this
+          function.
+          This function is also allowed to be called when the
+          distribution controller for the distribution channel
+          identified by <c><anno>DHandle</anno></c> is a port,
+          for the port owner to be able to call
+          <seemfa marker="erlang#dist_ctrl_put_data/2"><c>erlang:dist_ctrl_put_data/2</c></seemfa>
+          when the port is using
+          <seemfa marker="kernel:inet#setopts/2"><c>{deliver, term}</c></seemfa>
+          option.
```

Comment:
```suggestion
          This function is also allowed to be called when the
          distribution controller for the distribution channel
          identified by <c><anno>DHandle</anno></c> is a port.
          The data received by the port should in this case be delivered to
          the process identified by <c><anno>InputHandler</anno></c>
          which in turn should call
          <seemfa marker="erlang#dist_ctrl_put_data/2"><c>erlang:dist_ctrl_put_data/2</c></seemfa>.
```

Using the inet-driver is a special case of using a port as distribution controller. You may use any driver you want and the only requirement is that the data received is delivered to the runtime system via the input handler process.